src/app/tests/suites/certification/Test_TC_SU_5_1.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
+
+name: 46.11.1. [TC-SU-5.1] Verifying vendor specific OTA implementation on DUT
+
+PICS:
+    - MCORE.OTA.VendorSpecific
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic Information"
+    endpoint: 0
+
+tests:
+    - label:
+          "Step 1: Perform the OTA Update on DUT using vendor specific
+          mechanism. Verify that the DUT starts updating its software. Once the
+          update is finished, verify the SoftwareVersion attribute from the
+          Basic Information cluster on the DUT to match the version downloaded
+          for the software update."
+      verification: |
+          Perform the OTA Update on DUT using vendor specific mechanism.
+
+          After the OTA-transfer is finished successfully , verfiy the software version is update:
+
+          ./chip-tool  basicinformation read software-version 321 0
```

Comment:
This seems like a brand new manual test. We should not add manual tests anymore

Reply:
I will remove this script for now, we can re-visit this functionality further on.